odoo/tools/translate.py

```diff
@@ -863,12 +863,6 @@ def push_translation(module, type, name, id, source, comments=None):
         elif model=='ir.actions.report.xml':
             name = encode(record.report_name)
             fname = ""
-            if record.report_rml:
-                fname = record.report_rml
-                parse_func = trans_parse_rml
-                report_type = "report"
-            elif record.report_xsl:
-                continue
             if fname and record.report_type in ('pdf', 'xsl'):
```

Comment:
report_type xsl doesn't seem to exist, and pdf was removed by the PR, maybe this block should be removed as well?